index.html

```diff
@@ -6,28 +6,30 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link rel="stylesheet" href="style.css">
     <link href='http://fonts.googleapis.com/css?family=Roboto' rel='stylesheet' type='text/css'>
-    <title>Document</title>
+    <title>Portfolio Kazaneza</title>
 </head>
 <body>
 
-    
+    <header>
         <nav class="nav">
             <label class="logo">KAZANEZA GENTIL</label>
             <a href="#">
                 <img class="navImage" src="./Images/ic_menu.svg" alt="Navigation menu">
             </a>
         </nav>
+    </header>   
+        
     
-        <header class="container">
+        <section class="container">
             <div class="headline"> Hey there. <br/> I'm Kazaneza</div>
             <div class="title">I’am a Software Developer</div>
             <div class="introduction"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, on’t hestiate to contact me.</div>
 
             <div class="icons">
                 <div class="button twitter"><img  src="./images/Social Media.png" alt=""></div>
```

Comment:
[OPTIONAL] Instead of a static image for social icons, please make sure The list of icons uses a `ul` tag to group them, and `li` per icon; each icon has a link (`a` tag).

* Apart from being semantically correct, the `a` tag will let you connect your social networks with the icons, using href.

for example :point_down: 
```
<ul>
          <li> <a href="">social icon</a> </li>
          <li> <a href="">social icon</a> </li>>
          <li> <a href="">social icon</a> </li>
          <li> <a href="">social icon</a> </li>
         <li> <a href="">social icon</a> </li>
   </ul>
```